debian/rules

```diff
@@ -7,110 +7,23 @@ export DH_VERBOSE=1
 
 package=logitechmediaserver
 base=$(CURDIR)/debian/$(package)
-share=usr/share/squeezeboxserver/
 source=$(CURDIR)/../server
 
-varlib=var/lib/squeezeboxserver
+# main packaging script based on dh7 syntax
+%:
+	dh $@ 
 
+# Overrides for variances within Logitech Media Server file locations
+override_dh_perl:
+	# using hard-coded dependency in control file
 
-clean:
-	dh_testdir
-	dh_testroot
-	rm -rf ${base}
-
-install:
-	dh_testdir
-	dh_testroot
-	dh_installdirs
-
-	# Create directories needed.
-	install -d -m0755 ${base}/etc/default/
-	install -d -m0755 ${base}/etc/squeezeboxserver/
-	install -d -m0755 ${base}/etc/init.d/
-	install -d -m0755 ${base}/usr/share/perl5/
-	install -d -m0755 ${base}/${share}
-	install -d -m0755 ${base}/usr/share/lintian/overrides/
-	install -d -m0755 ${base}/usr/share/doc/squeezeboxserver/
-	install -d -m0755 ${base}/${varlib}/prefs
-	install -d -m0755 ${base}/${varlib}/cache
+override_dh_installdocs:
+	cp ${source}/License.txt $(CURDIR)/debian/copyright
+	dh_installdocs
 
+override_dh_install:
+	dh_install -X darwin -X freebsd -X MSWin -X sparc-linux -X PreventStandby -X errmsg
 	# Copy our server files to the appropriate Debian locations.
 	install -m0755 ${source}/slimserver.pl ${base}/usr/sbin/squeezeboxserver
 	install -m0755 ${source}/scanner.pl ${base}/usr/sbin/squeezeboxserver-scanner
 	install -m0755 ${source}/cleanup.pl ${base}/usr/sbin/squeezeboxserver-cleanup
-
-	cp -r ${source}/Slim ${base}/usr/share/perl5/
-	
-	# Copy our CPAN directory and locally modified perl modules
-	cp -r ${source}/CPAN ${base}/${share}
-	cp -r ${source}/lib ${base}/${share}
-
-	# Make sure we copy in the Bin directory
-	cp -r ${source}/Bin ${base}/${share}
-
-	# Remove non-Linux binary modules and binaries
```

Comment:
I don't see where in the new build configuration this would be handled? Are incompatible files no longer removed?